src/compas/robots/model/robot.py

```diff
@@ -507,6 +516,27 @@ def get_base_link_name(self):
         joints = self.get_configurable_joints()
         return joints[0].parent.link
 
+    @staticmethod
+    def _get_item_meshes(item):
```

Comment:
This function doesn't really belong here, but where instead?

Reply:
how about moving it to the `Geometry` class of the robot model `compas/robots/model/geometry.py`? (probably staying as a `staticmethod`)